hazelcast/src/test/java/com/hazelcast/query/impl/predicates/PredicatesTest.java

```diff
@@ -252,6 +253,12 @@ public void testILike() {
     }
 
     @Test
+    public void testILike_Id() {
+        ILikePredicate predicate = (ILikePredicate) ilike(ATTRIBUTE, "Java_World");
+        assertEquals(PredicateDataSerializerHook.ILIKE_PREDICATE, predicate.getId());
```

Comment:
hmm, I know what you mean, but referencing 6 would be a magic number that nobody would understand. This test is not very clever anyway...